src/test/run-make-fulldeps/coverage-reports/Makefile

```diff
@@ -87,7 +87,7 @@ endif
 	# Run it in order to generate some profiling data,
 	# with `LLVM_PROFILE_FILE=<profdata_file>` environment variable set to
 	# output the coverage stats for this run.
-	LLVM_PROFILE_FILE="$(TMPDIR)"/$@-%p.profraw \
+	LLVM_PROFILE_FILE="$(TMPDIR)"/$@.profraw \
```

Comment:
👍  I support the change on this line (removing pid from the non-doc-test run), even though it shouldn't make a difference in the result.